walnut/cmd/convert.go

```diff
@@ -235,7 +237,11 @@ func init() {
 
 	convertCmd.PersistentFlags().StringVarP(&extraVars, "extra-vars", "", "",
 		"extra vars")
-	_ = viper.BindPFlag("extra-vars", rootCmd.Flags().Lookup("static-config-uri"))
+	_ = viper.BindPFlag("extra-vars", rootCmd.Flags().Lookup("extra-vars"))
+
+	convertCmd.PersistentFlags().StringVarP(&taskNamePrefix, "task-name-prefix", "", "",
+		"prefix for task name fields")
+	_ = viper.BindPFlag("task-name-prefix", rootCmd.Flags().Lookup("task-name-prefix"))
```

Comment:
I believe these are getting a bit too messy, any better way to approach these?

Reply:
I don't think so. They can be bound to vars and such but you still have to register the flags with `viper`.